### DISCUSSION TITLE
Support for Mobile App?

### DISCUSSION THREAD

**Discussion by kerim** [vslinko/obsidian-outliner#81] [Archive]
I haven't yet used the closed beta mobile app, but seeing as this is very much based on keyboard commands, do you have plans to release a version that will work with a touch interface on mobile?

**Comment by vslinko** [comment]
I have thought about it. I want to support mobile, but it seems like I have to rewrite entire plug-in, because Obsidian mobile uses CodeMirror 6 when Obsidian desktop uses CodeMirror 5. 
So, maybe I’ll do that sometime.

**Comment by vslinko** [comment]
Archiving in favor of the discussion https://github.com/vslinko/obsidian-outliner/discussions/124
